scheduler/scheduler.go

```diff
@@ -17,10 +17,17 @@
 
 package scheduler
 
+import "os/exec"
+
 var scheduler schedulerInterface
 
 func init() {
-	scheduler = &cron{}
+	_, err := exec.LookPath("launchd")
```

Comment:
Hmm, given that the executable you actually run is called `launchctl`, wouldn't it make sense to look for that? (It's even defined: `const launchctl string = "launchctl"`

Reply:
Agreed!